examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -74,18 +66,12 @@ class GalleryAppState extends State<GalleryApp> {
       },
     );
 
-    if (showPreviewBanner)
-      home = new Banner(
-        message: 'PREVIEW',
-        location: BannerLocation.topRight,
-        child: home,
-      );
-
-    if (config.updateUrlFetcher != null)
+    if (config.updateUrlFetcher != null) {
```

Comment:
I'll ask him about it. I've been putting braces around multi-line statements. Maybe I'm a scofflaw :-)